### PR TITLE
Include LICENSE.md in sdist tarball

### PR DESCRIPTION
**Pull request by xhochy**
This ensures that the license is shipped in the sdist tarball on PyPI.

### CODE REVIEW

**Comment by jakirkham** [comment]
Think we can add [this to `setup.cfg`]( https://github.com/pypa/sampleproject/blob/d4ee05fdc03e848ed6e7065d8fe8e833a3c8c0b2/setup.cfg#L1-L4 )

File: setup.cfg [hunk 1]
```diff
@@ -1,3 +1,7 @@
+[metadata]
+# This includes the license file(s) in the wheel.
+license_files = LICENSE.md
```
Comment (jakirkham) [inline setup.cfg:3]: This should also work for `sdist`s

https://github.com/pypa/setuptools/pull/1536

**Comment by fritzo** [comment]
Hi @xhochy thanks for the fix.  Could you merge in master to get the fix from #599 that's causing the CI failure?

**Comment by xhochy** [comment]
> Hi @xhochy thanks for the fix. Could you merge in master to get the fix from #599 that's causing the CI failure? (I'm not sure how to do that as a maintainer from the github web ui)

Done.

**Review by fritzo — APPROVED** [review]

**Pull request merged**